server/graphql/v2/mutation/index.js

```diff
@@ -17,6 +17,7 @@ import paymentMethodMutations from './PaymentMethodMutations';
 import payoutMethodMutations from './PayoutMethodMutations';
 import transactionMutations from './TransactionMutations';
 import updateMutations from './UpdateMutations';
+import userMutations from './UserMutations';
```

Comment:
To be confirmed by @znarf, but I think the term "User" is banned in GQLV2. We rather refer to them as "Individuals"

Reply:
@znarf : Should we change this you think? 😄

Reply:
It's not a big problem because it's not user facing.

But it you ask me, it should not been named User but Account or Individual. There should be ideally no "User" in GraphQL API v2, it's confusing.

Reply:
Done. I've renamed those references. 😄